src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -256,8 +256,8 @@ public interface IDurableOrchestrationClient
         /// </summary>
         /// <param name="instanceId">The ID of the orchestration instance to query.</param>
         /// <param name="showHistory">Boolean marker for including execution history in the response.</param>
-        /// <param name="showHistoryOutput">Boolean marker for including input and output in the execution history response.</param>
```

Comment:
Oh, interesting. I didn't realize that `showHistoryOutput` was also documented to show the input...

Reply:
So I guess it was a bug that we never displayed the input even though we said we would. Since we never showed the output before, it should be safe for us to change the documentation here, and instead rely on `showInput`.